solidity-plugin/src/main/java/org/sonarsource/solidity/CognitiveComplexityVisitor.java

```diff
@@ -118,12 +121,41 @@ private boolean checkExpressionIncrementsComplexity(ExpressionContext exprList)
     return false;
   }
 
+  private static int countConditionalOperators(ExpressionContext expr) {
+    int count = 1;
+    for (ParseTree tree : expr.children) {
+      if (TerminalNode.class.isInstance(tree) && isAndOrOperator(tree)) {
+        count++;
+      }
+    }
+    return count;
+  }
+
+  @Override
+  public Token visitTerminal(TerminalNode node) {
+    if (isAndOrOperator(node)) {
+      complexity++;
+    }
+    return super.visitTerminal(node);
+
+  }
+
+  @Override
+  public Token visitStatement(StatementContext ctx) {
+    if (Utils.isTernaryExpression(ctx)) {
+      // ExpressionContext expression = Utils.countTernaryExpressionOperators(ctx);
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: This block of commented-out lines of code should be removed. ([squid:CommentedOutCodeLine](https://sonarcloud.io/organizations/stylianos-agapiou-sonarsource-gi/rules#rule_key=squid%3ACommentedOutCodeLine))

[See it in SonarCloud](https://sonarcloud.io/project/issues?id=org.sonarsource.solidity%3Asonar-solidity&pullRequest=22&issues=AWQ2SgNzYwZU9vvru9oh&open=AWQ2SgNzYwZU9vvru9oh)

[ISSUE_KEY]: AWQ2SgNzYwZU9vvru9oh